htmresearch/frameworks/clustering/sdr_clustering.py

```diff
@@ -90,10 +103,13 @@ def __init__(self,
                stableThreshold,
                minClusterSize,
                pointSimilarityThreshold,
+               pruningFrequency,
+               pruneClusters,
                fistClusterId=0):
 
     # Clusters
-    self._newCluster = Cluster(fistClusterId)
+    self._numPointsClustered = 0
```

Comment:
The name is a little confusing, as we are not adding every single input to the clusters. How about iteration or steps?